etca_asm/extensions/stack_and_functions.py

```diff
@@ -0,0 +1,86 @@
+from etca_asm.core import Extension, reject, oneof
+from etca_asm.base_isa import CONDITION_NAMES, validate_registers, build
+
+functions = Extension(2, "functions", "Stack and Functions")
+Register = tuple[int | None, int]
+
+REGISTERS = {
```

Comment:
These don't match the spec. v0 should be a2 and a0 and a1 should also be v0 and v1.